apps/filebrowser/src/filebrowser/api.py

```diff
@@ -129,4 +130,25 @@ def rename(request):
     raise Exception(_('The destination path "%s" already exists.') % dest_path)
 
   request.fs.rename(src_path, dest_path)
-  return HttpResponse(status=200)
\ No newline at end of file
+  return HttpResponse(status=200)
+
+@error_handler
+def content_summary(request, path):
+  path = _normalize_path(path)
+  response = {'summary': None}
```

Comment:
I'm now thinking we don't need to set it here. What if for all bad cases response is `{}` but for the good cases we send `response = {'summary': summary}`. (line 150)

Looks a bit more cleaner to handle in the UI, what do you think?